src/dispatch/case/views.py

```diff
@@ -145,6 +147,30 @@ def create_case(
     return case
 
 
+@router.post(
```

Comment:
Should we do this as a background task with background_tasks.add_task()?

Reply:
Done.